src/test/lib/storage/index/partial_hash/partial_hash_index_test.cpp

```diff
@@ -70,22 +75,20 @@ TEST_F(PartialHashIndexTest, IndexCoverage) {
 TEST_F(PartialHashIndexTest, EmptyInitialization) {
   EXPECT_THROW(PartialHashIndex(std::vector<std::pair<ChunkID, std::shared_ptr<Chunk>>>(), ColumnID{0}),
                std::logic_error);
-  auto empty_index = PartialHashIndex(DataType::String, ColumnID{0});
+  EXPECT_EQ(empty_index_impl->cbegin(), empty_index_impl->cend());
```

Comment:
Adding `PartialHashIndexTest` as a friend of `PartialHashIndex` allows the tests `SetUp()`-method to access the indeces protected funtions. If we want to use them inside the test cases, we would need to add each of them as a friend individually.
I don't think this would be a nice solution, or is there an easier way to do this?

Reply:
You could add utility functions to the test class, each calling the corresponding protected function, see https://github.com/hyrise/hyrise/blob/74727cfdb19b0c75d86bcef0007553f95b6ad8ec/src/test/lib/concurrency/transaction_manager_test.cpp#L11-L26.